### ISSUE TITLE
Inpainting

### ISSUE THREAD

**Issue by daniel1896** [LouisRouss/Diffusion-Based-Model-for-Colorization#1]
The results shown in the paper look really promising and I am interested in using it for inpainting.
What do I need to do/change to use this for inpainting? 
Is it a lot, because this would be my first project of this kind?

**Comment by LouisRouss** [comment]
Good afternoon,

For adapting this code to an inpainting task not much needs to be done, first obviously the dataloader will need to be changed accordingly to the dataset you use (with transformation such as resizing if needed, the implementation of masked images etc...). For the rest basically you should consider a similar role between B&W images/Masked images and Colored Images/Filled Images and modify the code accordingly (for exemple in the forward call of the unet network: *y* would be a tensor of noisy filled images and *x* a tensor of masked images).

About this repo: I haven't been able to train any network, the computation power needed is beyond my capabilities, it was more like a coding exercise for me. If you can train it I would love to read your feedback.

**Comment by daniel1896** [comment]
Thank you, that helps a lot. I will see if I can make it work.
